p/invg/products.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+module.exports = [
+	// https://github.com/public-transport/hafas-client/issues/93#issuecomment-437868265
+	{
+		id: 'bus',
+		mode: 'bus',
+		bitmasks: [1],
+		name: 'Bus',
+		short: 'Bus',
+		default: true // the other `bus` has `false`
+	},
+	{
+		id: 'express-train',
```

Comment:
*[minor]* Are you sure the naming is appropriate here? If I understand the profile correctly this includes intercity and eurocity trains, which are faster than regional, but not really high speed.

Reply:
I didn't think much about this name, I may even have copied it from another profile. "express" here rather meant "only stopping at some stations", but we can change it to something else.